package.json

```diff
@@ -45,14 +45,15 @@
     "console-stamp": "~0.1.1"
   },
   "devDependencies": {
-    "supertest": "~0.8.0",
-    "should": "~1.3.0",
-    "grunt-contrib-watch": "~0.5.3",
-    "grunt-contrib-jshint": "~0.6.4",
-    "grunt-nodemon": "~0.1.1",
+    "dotenv": "^4.0.0",
```

Comment:
I believe this should be part of your applications dependencies. Please fix this.